content/en/getting_started/synthetics/api_test.md

```diff
@@ -86,3 +108,4 @@ Datadog also has an [APM integration with Synthetic Monitoring][5] which allows
 [3]: /api/v1/synthetics/#create-or-clone-a-test
 [4]: /integrations/
 [5]: /synthetics/apm/
+[6]: /getting-started/synthetics/private_location/
```

Comment:
```suggestion
```
You don't need this line for `further_reading` links. All links listed in the top-matter (lines 1-21 in this file) will be inserted where the `partial` call is (line 104). This section (lines 106-110) is strictly for the in-line links in the page text.